.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
```

Comment:
- [ ] What happens here if the trip is not successfully destroyed?  Add some error handling:

![image](https://github.com/Swarzstein/final-capstone/assets/57421823/c38d1da1-762a-4291-8e16-27913670acf4)